Blink/TermController.swift

```diff
@@ -358,6 +357,10 @@ extension TermController: TermDeviceDelegate {
     return self
   }
   
+  public func xCallbackLineSubmitted(_ line: String!, _ successUrl: URL? = nil) {
```

Comment:
I think we can remove optional on line

Reply:
If coming from `Shortcuts.app` we have a `x-success`, `x-error` & `x-cancel` URL to return back to the app. If for example you launch the same `x-callback-url` from `Safari.app` you won't get those URLs as there's no need to return back to Safari once the execution has finished. Hence, the need for an optional URL to return back to the original app if it was provided.

Reply:
Sorry, i'm talking about  implicitly unwrapped optional `String!` it could are replaced just with `String`.